tests/python/topi/python/test_topi_dense.py

```diff
@@ -75,6 +75,8 @@ def check_device(device, ctx):
             b = tvm.nd.array(b_np, ctx)
             c = tvm.nd.array(c_np, ctx)
             d = tvm.nd.array(np.zeros(get_const_tuple(D.shape), dtype=dtype), ctx)
+            print(tvm.lower(s, [A, B, C, D], simple_mode=True))
+            exit()
```

Comment:
```suggestion
```

Reply:
oh whoops, this is a remnant of some unrelated debugging i was doing. Thanks for catching it.